src/App.tsx

```diff
@@ -1,19 +1,119 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  FormEvent,
+  useCallback,
+  useEffect,
+  useRef,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
+import { TodoList } from './components/Auth/TodoList';
+import { Errormessage } from './components/Auth/Errormessage';
+import { Footer } from './components/Auth/Footer';
+import { Todo } from './types/Todo';
+import { FilterType } from './types/SortType';
+import { getTodos, createTodos, deleteTodos } from './api/todos';
+import { ErrorMessage } from './types/Error';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const [todos, setTodos] = React.useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = React.useState('');
+  const [filterType, setFilterType] = React.useState('all');
```

Comment:
```suggestion
  const [filterType, setFilterType] = React.useState<FilterType>(FilterType.All);
```
better type state with enum